docs/index.md

```diff
@@ -178,6 +178,21 @@ plugins:
           ...
 ```
 
+You can also define the category-specific settings in a YAML file. To do that, you need to add a new default key `config_filename` in the plugin configuration:
+
+```yaml title="mkdocs.yml"
+plugins:
+  - blogging:
+      config_filename: .pages
+```
+
+
+!!! notes "Awesome pages"
+    You can use the same file as the [awesome pages plugin](https://github.com/lukasgeiter/mkdocs-awesome-pages-plugin).
+
+Moreover, you can also automatically create blog pages for each root directory. Theses pages will be named after the directory name. To do that, you need to add or set the key `use_root_dirs` to `true` in the plugin configuration.
```

Comment:
Would it be better to place this line in the "Of all the options mentioned above, these deserve special attention..." section below, since it's actually about a global configuration?